tests/spark/test_spark_model_export.py

```diff
@@ -59,12 +61,19 @@ def spark_context():
              value='ml.combust.mleap:mleap-spark-base_2.11:0.12.0,'
                    'ml.combust.mleap:mleap-spark_2.11:0.12.0')
     conf.set(key="spark_session.python.worker.reuse", value=True)
-    spark = pyspark.sql.SparkSession.builder\
-        .config(conf=conf)\
-        .master("local-cluster[2, 1, 1024]")\
-        .getOrCreate()
-    sc = spark.sparkContext
-    return sc
+    max_tries = 3
```

Comment:
Seems like SparkContext initialization can flake while attempting to fetch mleap libraries (specified as part of the Spark conf [here](https://github.com/mlflow/mlflow/pull/2373/files#diff-9b8029c7d29cba0c31d59382d11e876dR60)), so added retries